pebblo/reports/html_to_pdf_generator/generator_functions.py

```diff
@@ -1,17 +1,26 @@
+"""
+Defines functions to generate PDF from HTML for respective renderers
+"""
+
+import os
 from weasyprint import HTML, CSS
 from xhtml2pdf import pisa
-import os
+
 
 # Creates PDF from template using weasyprint
-def weasyprint_pdf_converter(sourceHtml, outputPath, searchPath):
+def weasyprint_pdf_converter(source_html, output_path, search_path):
+    """PDF generator function for weasyprint renderer"""
     base_url = os.path.dirname(os.path.realpath(__file__))
-    htmldoc = HTML(string=sourceHtml, base_url=base_url)
-    return htmldoc.write_pdf(target=outputPath, stylesheets=[CSS(searchPath + '/index.css')])
+    html_doc = HTML(string=source_html, base_url=base_url)
+    return html_doc.write_pdf(
+        target=output_path, stylesheets=[CSS(search_path + "/index.css")]
+    )
+
 
 # Creates PDF from template using xhtml2pdf
-def xhtml2pdf_pdf_converter(sourceHtml, outputPath, searchPath):
-    resultFile = open(outputPath, "w+b")
-    pisaStatus = pisa.CreatePDF(src=sourceHtml,           
-        dest=resultFile)
-    resultFile.close()
-    return pisaStatus.err
\ No newline at end of file
+def xhtml2pdf_pdf_converter(source_html, output_path, search_path):
```

Comment:
Remove unused parameter `search_path` or use a special variable name like `_` for unused parameter

Reply:
Resolved